modules/cluster-logging-updating-logging.adoc

```diff
@@ -32,6 +32,21 @@ If your cluster logging version is prior to 4.4, you must upgrade cluster loggin
 
 * Back up your Elasticsearch and Kibana data.
 
+* If your internal Elasticsearch instance uses persistent volume claims (PVC), the PVCs must contain a `logging-cluster:elasticsearch` label. If you are upgrading from an {product-title} version prior to version 4.4.30, you must manually add the label to the Elasticsearch PVCs or the PVCs get removed by the garbage collection process and recreated by the Elasticsearch operator during the upgrade. After {product-title} 4.4.30, the Elasticsearch operator automatically adds the label when it creates the PVC. 
++
+For example:
```

Comment:
@mburke5678 how about using `oc` command here?
```
oc label pvc --all -n openshift-logging logging-cluster=elasticsearch
```
or add label to PVC one by one:
```
oc label pvc $pvc-name -n openshift-logging logging-cluster=elasticsearch
```

Reply:
@QiaolingTang Am I doing something wrong? I cannot label the PVCs. 
If I use the `oc label pvc` command, the PVC gets labeled, but it is immediately removed.
If I edit the PVC in the console or CLI, I get an error:

`PersistentVolumeClaim in version "v1" cannot be handled as a PersistentVolumeClaim: v1.PersistentVolumeClaim.ObjectMeta: v1.ObjectMeta.Labels: ReadString: expects " or n, but found t, error found in #10 byte of ...|csearch":true}},"spe|..., bigger context ...|logging-cluster":"elasticsearch","elasticsearch":true}},"spec":{"accessModes":["ReadWriteOnce"],"res|...`

Reply:
@mburke5678 which version of EO did you deployed? I tried several times in my cluster and found if the EO has the bug fix, after adding labels to the PVCs, the EO rewrites the PVC label to `logging-cluster=elasticsearch`. If scale down the EO, then the labels can be added successfully

Reply:
@QiaolingTang I used a 4.6 and 4.4.30 (which has the fix). I didn't realize that the EO rewrites the labels. I will try an earlier version. Thank you.